example/Index.re

```diff
@@ -1,11 +1,24 @@
 open Express;
 
-let __dirname: option string = [%bs.node __dirname];
+let app = express ();
 
-let app = Express.express ();
+App.useOnPath app path::"/" @@ Middleware.fromF (fun _ _ next => {
+  Js.log "Request received"; 
+     /* This will be printed for every request */
+  next Js.undefined [@bs]
```

Comment:
@bobzhang can comment on the feasibility of avoiding `Curry` without the [@bs] attribute.

I am not sure what the consensus is when doing binding should we use [@bs] or not. The attribute itself is documentation since it is part of the BS language.